### PR TITLE
Add Google Analytics

### LINKED ISSUE #37: Add Google Analytics
## Expected Behavior

Tracking with Google Analytics

### PR DESCRIPTION
**Pull request by lizgene**
## Why?

Let's see who are the peeps on our site. 

Resolves #37 

## What?

- Adds GA tracking code.
- Adds a default layout to include this tracking code easily on all pages.
- Removes duplicate layouts.

## Screenshots (if appropriate)

image

I like Omnibug for checking out analytics params:

image

## Questions/TODOs:

- Anyone know a good way to see if this is in dev/prod so we can use a dev tracking code for local development?

### CODE REVIEW

**Comment by dashosaur** [comment]
You can use `{% if jekyll.environment == "production" %}` according to [this documentation page](https://jekyllrb.com/docs/configuration/)! `development` is the default value and GitHub automatically sets the it to `production` for us.

File: _includes/google-analytics.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-97614019-1', 'auto');
+  ga('send', 'pageview');
+
```
Comment (dashosaur) [inline _includes/google-analytics.html]: nit: extra line break

**Review by dashosaur — COMMENTED** [review]
Looks good. I'll approve after you add the environment check.

**Comment by lizgene** [comment]
@dashland Added the check for `production` and removed the extra spaces.

**Review by dashosaur — APPROVED** [review]
Looks good!

**Pull request merged**